setup_ubuntu.sh

```diff
@@ -0,0 +1 @@
+apt-get install python-pyaudio python-wxGlade
```

Comment:
It's enough to have python-wxgtk2.8 instead of python-wxGlade.
See https://morevnaproject.org/papagayo/ ^__^